### PR TITLE
[Jetpack AI] Disclaimer

### PR DESCRIPTION
**Pull request by artpi**
This adds a disclaimer to make it clear if you can trust Jetpack AI.
Zrzut ekranu 2023-01-26 o 19 15 18

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Use any of the AI blocks (search for `magic`)
* Click on the sidebar
* See disclaimer

### CODE REVIEW

**Review by jeherve — COMMENTED** [review]
Do you think we could run this by Editorial? I'm not sure we should be using the brand "Jetpack" here, since that's something that may not mean much to WordPress.com simple and WoA site owners who will first see this?

I'll add the label to this PR to flag it with Editorial.

File: projects/plugins/jetpack/extensions/blocks/ai-image/index.js [hunk 1]
```diff
@@ -30,6 +30,12 @@ export const settings = {
 					'jetpack'
 				) }
 			</p>
+			<p>
```
Comment (DavidRothstein) [inline projects/plugins/jetpack/extensions/blocks/ai-image/index.js:33]: Wondering if all or part of this paragraph should be highlighted (`<strong>`, for example) to make it stand out more?

File: projects/plugins/jetpack/extensions/blocks/ai-image/index.js [hunk 2]
```diff
@@ -30,6 +30,12 @@ export const settings = {
...
+				{ __(
+					'The content is generated by OpenAI. Jetpack does not endorse, nor vouch for accuracy of the output. Please use your best judgement as you are the ultimate arbiter of what you publish on your site.',
```
Comment (DavidRothstein) [inline projects/plugins/jetpack/extensions/blocks/ai-image/index.js:35]: Would the first sentence be better something like the following (to provide more context for people who don't know what OpenAI is)?

>The content is generated by a computer using OpenAI and has not been reviewed by a human.

**Review by DavidRothstein — COMMENTED** [review]
Looks good to me overall!  I agree with the point above about referencing "Jetpack", and left a couple other small comments also.

**Comment by millerf** [comment]
Is "arbiter" a real word?

**Comment by artpi** [comment]
There is a companion diff that does some amount of moderation: D99473-code

**Pull request closed without merging**